nixos/modules/services/misc/klipper.nix

```diff
@@ -69,6 +67,26 @@ in
           for supported values.
         '';
       };
+
+      firmware = mkOption {
```

Comment:
Seems like this should be called `firmwares`?

Reply:
Done